api-inference-community/api_inference_community/validation.py

```diff
@@ -308,7 +309,25 @@ def normalize_payload_image(bpayload: bytes) -> Tuple[Any, Dict]:
     return img, {}
 
 
+AUDIO_EXTENSIONS = {".mp3", ".wav", ".flac", ".mp4", ".webm", ".aac"}
```

Comment:
@Narsil this was the source of the problem. when splitting the filename and extension with

```
filename, ext = os.path.splitext(bpayload)
```

the `ext` variable includes the dot `"."` so we were never actually reading the audio file.

i've reverted my previous change so think we can now re-deploy this to test it out with `superb`!

Reply:
Thanks !